css-brochure-form/styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+fieldset {
+  max-width: 400px;
+  height: 700px;
+  margin-right: auto;
+  margin-left: auto;
+  border: none;
+}
+
+legend {
+  text-align: center;
+  color: rgba(236, 93, 87, 255);
+  font-size: 1.5em;
+  font-weight: lighter;
+  letter-spacing: 2px;
+}
+
+input,
+select {
+  border-radius: 7px;
+  display: inline-block;
+  justify-content: stretch;
+  border-color: lightgray;
+  height: 70px;
+  width: 900px;
+  font-size: 2rem;
+  padding-left: 15px;
+}
+
+select {
+  background-color: #f3f3f3;
+}
+
+input {
+  margin: 0 0 50px 0;
+}
+
+button {
+  justify-content: flex-end;
+  font-weight: 550;
+  height: 70px;
+  width: 200px;
+  border-radius: 35px;
+  margin-top: 20px;
+  font-size: 1.9rem;
+  background-color: rgba(236, 93, 87, 255);
+  color: #fff;
+  border: none;
+}
+
+div:last-child {
+  float: right;
+}
+
+label {
+  font-weight: 550;
+  font-size: 1.3em;
+  word-spacing: 1px;
+  display: inline-block;
+  margin-bottom: 5px;
+  padding-left: 15px;
+}
+
+/* Since labels default as an inline element, they thereby do not come with a top and bottom margin. As a result,
+setting top and bottom margins on them won't have any effect. Their display must be reset to some property similar to the block, so
+it then acquires the top and bottom margin. */
```

Comment:
Very correct! Rather than including notes like this directly in your code though, I would collect these elsewhere. In general, comments in your code should be kept to a minimum. They are usually not needed, and just add clutter to the code.